src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -43,10 +44,7 @@ const StyledCheckBoxContainer = styled.label`
   flex-direction: row;
   align-items: center;
   user-select: none;
-  ${props =>
-    props.fill
-      ? fillStyle() && `justify-content: space-between;`
-      : 'width: fit-content;'}
+  ${props => (props.fill ? fillStyle() : 'width: fit-content;')}
```

Comment:
This is a nice clean up.. I would maybe add a comment to `fillstyle` that it includes the `justify-content: space-between;` just incase it ends up being used again..